### PR TITLE
feat: added copyToClipboard for all address

### PR DESCRIPTION
**Pull request by Teja2045**

### CODE REVIEW

File: frontend/src/components/CopyToClipboard.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
+import ContentCopyOutlined from "@mui/icons-material/ContentCopyOutlined";
+import { Box, Chip, IconButton, Tooltip } from "@mui/material";
+import React from "react";
+import { useDispatch } from "react-redux";
+import { copyToClipboard } from "../utils/clipboard";
+import { shortenAddress } from "../utils/util";
+
+export const CopyToClipboard = (props) => {
+  const { message } = props;
+  const toolTipEnabled = props.toolTipEnabled;
```
Comment (aleem1314) [inline frontend/src/components/CopyToClipboard.js:10]:
```suggestion
  const { message, enableTooltip } = props;
```

**Review by aleem1314 — APPROVED** [review]
left one comment. Otherwise LGTM

File: frontend/src/components/CopyToClipboard.js (hunk 1 above)
Comment (Teja2045) [reply frontend/src/components/CopyToClipboard.js:10]: const { message, enableTooltip } throws error if enableTooltip is not in props

Comment (Teja2045) [reply frontend/src/components/CopyToClipboard.js]: done

**Pull request merged**